nimble-newts/askgrieves/chatbot/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class Chatbot(models.Model):
+    name = models.CharField(max_length=500)
+
+    def __str__(self):
+        return f"name={self.name}"
```

Comment:
This looks like somewhat of an attempt at a repr implementation, I can't see `name=abc` being useful otherwise.

Maybe this should be implemented as a repr similar to:

```py
In [4]: class Chatbot:
   ...:     def __init__(self, name):
   ...:         self.name = name
   ...:     def __repr__(self):
   ...:         return f"Chatbot(name={repr(self.name)})"
   ...:
   ...:

In [5]: Chatbot("abc")
Out[5]: Chatbot(name='abc')
```